src/wireviz/Harness.py

```diff
@@ -171,15 +169,13 @@ def create_graph(self) -> Graph:
                      f'{cable.gauge} {cable.gauge_unit}{awg_fmt}' if cable.gauge else None,
                      '+ S' if cable.shield else None,
                      f'{cable.length} m' if cable.length > 0 else None,
-                     cable.color, '<!-- colorbar -->' if cable.color else None],
+                     cable.color, html_colorbar(cable.color)],
                     '<!-- wire table -->',
+                    [html_image(cable)],
+                    [html_caption(cable)],
```

Comment:
`html_caption(cable.caption)` is nicer than passing along the entire cable object for a single property

Reply:
The function need more than one attribute from the node.